utils/script-native.c

```diff
@@ -66,6 +68,13 @@ static int load_native_api_funcs(const char *so_pathname)
 	}
 	pr_dbg("%s is loaded\n", so_pathname);
 
+	INIT_NATIVE_API_FUNC(uftrace_script_version);
+	if (SCRIPT_VERSION != __uftrace_script_version()) {
+		pr_warn("incompatible script versions (uftrace %d != module %d)\n",
+			SCRIPT_VERSION, __uftrace_script_version());
```

Comment:
Well.. I think we should have backward compatibility so old version of script can still be run.

Reply:
Yes, it's right. But that's why I made the major and minor version to handle this before. If we want to keep the backward compatibility then we must not change the layout of struct, but we can only append the data to the struct in `include/script.h`.